pkg/openthread/Makefile

```diff
@@ -0,0 +1,34 @@
+PKG_NAME=openthread
+PKG_URL=https://github.com/openthread/openthread.git
+PKG_VERSION=fbfd76a990b81f007957e1bd774e51bce742e53e
+PKG_BUILDDIR ?= $(BINDIRBASE)/pkg/$(BOARD)/$(PKG_NAME)
+
+$(info Compile OpenThread for FTD device)
+OPENTHREAD_ARGS+= --enable-cli-app=ftd --enable-application-coap
+
+$(info $$OPENTHREAD_ARGS is [${OPENTHREAD_ARGS}])
+
+.PHONY: all
+
+all: git-download
+	cd $(PKG_BUILDDIR) && PREFIX="/" ./bootstrap
+	cd $(PKG_BUILDDIR) && ./configure \
+		CPP="$(CPP)" CC="$(CC)" CXX="$(CXX)" OBJC="" OBJCXX="" AR="$(AR)" RANLIB="$(RANLIB)" NM="$(NM)" \
+		STRIP="$(STRIP)" \
+		CPPFLAGS="-fdata-sections -ffunction-sections -Os -g  $(CFLAGS_CPU) " \
```

Comment:
`-fdata-sections -ffunction-sections -Os -g  $(CFLAGS_CPU)` can be factorized between CPPFLAGS, CFLAGS, CXXFLAGS and LDFLAGS

is -g really required ?